string cmd
int32 trials
float32 distance
float32 distance_tolerance
float32 effort
float32 effort_tolerance
int32[] fingertip_profile0
int32[] fingertip_profile_min0
int32[] fingertip_profile_max0
int32[] fingertip_profile1
int32[] fingertip_profile_min1
int32[] fingertip_profile_max1
float32 stiffness
float32 stiffness_threshold
---
int32 result
float32[] distance
float32[] effort
float32 velocity
int32[] fingertip_profile0
int32[] fingertip_profile1
float32[] stiffness
int32[] force_peak0
int32[] force_steady0
int32[] force_peak1
int32[] force_steady1
float32[] distance_compressed
float32[] distance_peak
float32[] time
float32[] time_to_first
float32[] time_to_return